.github/workflows/cd-dicomservice.yml

```
@@ -0,0 +1,165 @@
+        required: true
+
+env:
+  BALLERINA_VERSION: 2201.12.10
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      JAVA_OPTS: -Xmx4G
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)